src/Compatibility/ApiDiff/Microsoft.DotNet.ApiDiff.Tool/Program.cs

```diff
@@ -0,0 +1,209 @@
+
+        Option<string> optionBeforeAssembliesFolderPath = new(["--before", "-b"])
+        {
+            Description = "The path to the folder containing the old (before) assemblies.",
+            Arity = ArgumentArity.ExactlyOne,
+            IsRequired = true
+        };
+
+        Option<string> optionBeforeRefAssembliesFolderPath = new(["--refbefore", "-rb"])
+        {
+            Description = "The path to the folder containing the old (before) reference assemblies.",
+            Arity = ArgumentArity.ExactlyOne,
+            IsRequired = false
+        };
+
+        Option<string> optionAfterAssembliesFolderPath = new(["--after", "-a"])
+        {
+            Description = "The path to the folder containing the new (after) assemblies.",
+            Arity = ArgumentArity.ExactlyOne,
+            IsRequired = false
+        };
+
+        Option<string> optionAfterRefAssembliesFolderPath = new(["--refafter", "-ra"])
+        {
+            Description = "The path to the folder containing the new (after) reference assemblies.",
+            Arity = ArgumentArity.ExactlyOne,
+            IsRequired = false
+        };
+
+        Option<string> optionOutputFolderPath = new(["--output", "-o"])
+        {
+            Description = "The path to the output folder.",
+            Arity = ArgumentArity.ExactlyOne,
+            IsRequired = true
+        };
+
+        Option<string> optionBeforeFriendlyName = new(["--beforeFriendlyName", "-bfn"])
+        {
+            Description = "The friendly name to describe the 'before' assembly.",
+            Arity = ArgumentArity.ExactlyOne,
+            IsRequired = true
+        };
+
+        Option<string> optionAfterFriendlyName = new(["--afterFriendlyName", "-afn"])
+        {
+            Description = "The friendly name to describe the 'after' assembly.",
+            Arity = ArgumentArity.ExactlyOne,
+            IsRequired = true
+        };
+
+        Option<string> optionTableOfContentsTitle = new(["--tableOfContentsTitle", "-tc"])
+        {
+            Description = "The title of the markdown table of contents file that is placed in the output folder.",
+            Arity = ArgumentArity.ExactlyOne,
+            IsRequired = true
+        };
+
+        Option<string[]?> optionAssembliesToExclude = new(["--assembliesToExclude", "-eas"], () => null)
+        {
+            Description = "Assemblies to exclude from the diff.",
+            Arity = ArgumentArity.ZeroOrMore,
+            IsRequired = false,
+        };
+
+        Option<string[]?> optionAttributesToExclude = new(["--attributesToExclude", "-eattrs"], () => null)
+        {
+            Description = "Attributes to exclude from the diff.",
+            Arity = ArgumentArity.ZeroOrMore,
+            IsRequired = false
+        };
+
+        Option<string[]?> optionApisToExclude = new(["--apisToExclude", "-eapis"], () => null)
+        {
+            Description = "APIs to exclude from the diff.",
```

Comment:
Same question as above - what's the format?

Reply:
Will do the same for this.